ansible_runner/runner_config.py

```diff
@@ -242,6 +242,13 @@ def prepare_env(self):
         self.job_timeout = self.settings.get('job_timeout', None)
         self.pexpect_timeout = self.settings.get('pexpect_timeout', 5)
 
+        self.process_isolation = self.settings.get('process_isolation', False)
+        self.process_isolation_executable = self.settings.get('process_isolation_executable', None)
+        self.process_isolation_path = self.settings.get('process_isolation_path', None)
```

Comment:
We have to default this at runtime, see https://github.com/ansible/ansible-runner/blob/c578377e12ee7fb6f788e493c8a96faa8d922cb4/ansible_runner/runner_config.py#L372

This is why it is None here, but documented as `bwrap`